editor.planx.uk/src/pages/FlowEditor/components/Settings/ServiceSettings/FlowStatus/index.tsx

```diff
@@ -104,28 +102,25 @@ const FlowStatus = () => {
         </Typography>
       </SettingsSection>
       <SettingsSection background>
-        <FormControlLabel
-          label={statusForm.values.status}
-          sx={{
-            marginBottom: 0.5,
-            [`& .${formControlLabelClasses.label}`]: {
-              fontWeight: FONT_WEIGHT_BOLD,
-              textTransform: "capitalize",
-              fontSize: 19,
+        <Switch
+          label={statusForm.values.status as Capitalize<string>}
+          formControlLabelProps={{
+            sx: {
+              marginBottom: 0.5,
+              [`& .${formControlLabelClasses.label}`]: {
+                fontWeight: FONT_WEIGHT_BOLD,
+                textTransform: "capitalize",
+                fontSize: 19,
+              },
```

Comment:
Switches in these forms are visually different - the label is also really the title of the card.

We could make this a variant, or could attempt to standardise here, but this seemed the best way to handle this initially.